plugins_examples/archive/archive.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"regexp"
+)
+
+func main() {
+	s := bufio.NewScanner(os.Stdin) 
+	for s.Scan() {
+		line := s.Text()
+		if strings.Contains(line, "PRIVMSG") == true {
+			skip := false;
+
+	    		re := regexp.MustCompile("https?\\:\\/\\/[[:^space:]]*")
+	    		match := re.FindString(line)
+
+			if match != "" && match != "https://arch.cccp.io/urls.txt" {
+	    			fr, err := os.Open("/var/www/html/arch/urls.txt")
```

Comment:
If i understand correctly, this line suggests the cabot instance that runs the plugin is local to your web server. If i want to run your plugin, i need to change this destination, and then find a way to upload to the web server (SSH with `command` in a cron can be enough, or a REST API :)).

Configuration variables could be at the beginning of the file:
- dest file
- whitelist array
- backlist array
- ...

Reply:
destFile is now a variable.